task05/models/userModel.js

```diff
@@ -0,0 +1,46 @@
+const { v4: uuidv4 } = require('uuid');
+let users = require('../models/users.db');
+
+function getAll() {
+  return new Promise((resolve, reject) => {
+    resolve(users);
+  });
```

Comment:
FYI: You can simplify it.

```suggestion
  return Promise.resolve(users)
```